src/components/filter/Filter.tsx

```diff
@@ -156,6 +152,12 @@ const styles = StyleSheet.create({
   icon: {
     paddingLeft: normalize(8)
   },
+  noPadding: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    paddingLeft: 0,
+    paddingRight: 0
```

Comment:
`padding: 0` wasn't fine?

Reply:
I tried but unfortunately it doesn't work. There is a padding:normalize(14) value for the wrapper. I cannot overwrite it.

Reply:
did you inspected the element? sometimes there are `paddingTop`, `paddingBottom`, `paddingLeft` and `paddingRight` set by something else, which cannot be overridden with just `padding`, so the explicit override with each value is needed. is this the case here?

Reply:
this part of the code has been reverted

Reply:
It is possible to overwrite the `padding` by using only the small prop for the `padding` in the wrapper component. I added `View` instead of `Wrapper` component and updated the style.